src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
Because the session IDs are different. For each websocket connection, the session ID will be different. There's no `setSessionID()` function and `init()` is called only once.

Also I make an array of users connected with their individual `SessionInstance` objects :
```
[
 0 => [
  'name' => 'duncan'
  'session' => SessionInstance
 ]
 1 => [
  'name' => 'subin',
  'session' => SessionInstance
 ]
]
```

I use Ratchet PHP library. The websocket listening happens in an infinite while loop.